app/controllers/revision_feedback_controller.rb

```diff
@@ -3,19 +3,54 @@
 require "#{Rails.root}/lib/importers/revision_score_importer"
 
 class RevisionFeedbackController < ApplicationController
+  before_action :initialize_ores
+
   def index
-    @wiki = Wiki.find_by(language: 'en', project: 'wikipedia')
-    @revision = Revision.find_by(mw_rev_id: params[:rev_id], wiki_id: @wiki.id)
-    set_new_revision unless @revision
-    @feedback = RevisionFeedbackService.new(@revision).feedback
+    set_revision_id
+    unless @revId.nil?
+      features = ores_features(@revId)
+      @feedback = RevisionFeedbackService.new(features).feedback
+    end
   end
 
   private
 
-  def set_new_revision
-    @revision = Revision.new(wiki_id: @wiki.id, mw_rev_id: params[:rev_id], mw_page_id: 0)
-    @revision.save!
-    RevisionScoreImporter.new.update_revision_scores([@revision])
-    @revision.reload
+  def initialize_ores
+    @wiki = Wiki.find_by(language: 'en', project: 'wikipedia')
+    @ores_api = OresApi.new(@wiki)
+  end
+
+    def ores_features(revId)
```

Comment:
This and the related methods duplicate a lot of logic from `RevisionScoreImporter`. Instead, we should probably just add a method to `RevisionScoreImporter` that fetches the ores data for a single rev_id without saving anything to the database. It looks like that just means moving this method to `RevisionsScoreImporter` (maybe as `#fetch_ores_features_for_revision_id`) and then call that instead of having the controller need to know about using and parsing responses from the ORES api.